### PR TITLE
support echarts3.0

### PR DESCRIPTION
**Pull request by HelloJasonZhang**
I have add echarts3.0 feature, but Only support  'dark','macarons','infographic' theme.

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -128,7 +128,8 @@ angular.module('angular-echarts.util', []).factory('util', function () {
 
                 // pie chart need special radius, center config
                 conf.center = config.center || ['40%', '50%'];
-                conf.radius = config.radius || '60%';
+                conf.radius = config.radius || ['40%', '55%'];
```
Comment (springuper) [inline src/util.js]: why use an array instead?

File: src/theme/shine.js [hunk 2]
```diff
@@ -5,218 +5,13 @@
...
-            detail: {
-                textStyle: {       // 其余属性默认使用全局文本样式，详见TEXTSTYLE
-                    color: 'auto'
-                }
-            }
-        },
-
-        textStyle: {
-            fontFamily: '微软雅黑, Arial, Verdana, sans-serif'
+    var colorPalette = ['#d87c7c','#919e8b', '#d7ab82',  '#6e7074','#61a0a8','#efa18d', '#787464', '#cc7e63', '#724e58', '#4b565b'];
```
Comment (springuper) [inline src/theme/shine.js]: why change color palette?

File: index.html [hunk 3]
```diff
@@ -32,12 +32,13 @@
 <div class="container">
 
     <h2 class="page-header">Angular Echarts <small>Angular bindings for baidu echarts</small></h2>
-
+
```
Comment (springuper) [inline index.html]: please delete this empty line

File: docs/docs.js [hunk 4]
```diff
@@ -34,12 +34,12 @@
     };
 
     function updateData($interval) {
-        $interval(function () {
+/*        $interval(function () {
```
Comment (springuper) [inline docs/docs.js]: why comment these lines?

**Comment by wangshijun** [comment]
Please resolve conflict first @HelloJasonZhang

**Comment by wangshijun** [comment]
Supports with echarts 3.0 will bring some breaking changes, can you port themes to the new version

**Comment by orangebook** [comment]
Yes , that is right

**Comment by talatbaig** [comment]
any update on when this will be live?

**Comment by wangshijun** [comment]
echarts 3.4 is now supported! please try v1.0.1

**Pull request closed without merging**